scraper.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
 require 'pry'
-require 'scraped'
 require 'scraperwiki'
+require_relative 'lib/members_page'
+require_relative 'lib/member_page'
+require_relative 'lib/noble_page'
 
 require 'open-uri/cached'
 OpenURI::Cache.cache_path = '.cache'
```

Comment:
If you're going to turn on archiving in a separate PR, it would be good for there to be an open issue for it (possibly linked in a comment)